iroha/src/queue.rs

```diff
@@ -49,7 +51,10 @@ impl Queue {
         }
     }
 
-    pub fn pop_pending_transactions(
+    /// Gets at most `maximum_transactions_in_block` number of transaction, but does not drop them out of the queue.
```

Comment:
Maybe add an explanation explaining why you don't remove them from the queue here.